creusot/src/translation/specification.rs

```diff
@@ -103,6 +116,10 @@ impl ContractClauses {
             let term = ctx.term(var_id).unwrap().clone();
             out.variant = Some(term);
         };
+        log::trace!("no_panic: {}", self.no_panic);
```

Comment:
note: I would like to see us upgrade to `tracing` for our logs, it provides more contextual / structured information especially when dealing with deep callgraphs.